autoload/vim_ai_config.vim

```diff
@@ -1,10 +1,12 @@
 let s:plugin_root = expand('<sfile>:p:h:h')
+let llm_endpont = exists('$LOCAL_LLM_URL') ? expand('$LOCAL_LLM_URL') : "https://api.openai.com"
```

Comment:
As this seems to be a helper variable, perhaps add a quick explanation why it is declared global instead of script local like the preceding ones? Just wondering, as it seems hard to define it as a user without risking it to be overwritten, since this assignment does not check for an existent variable of the same name.

Reply:
Actually that was due to my noob-level vimscript skills.
I will make the variable script local.
Thanks for the comment 👍